core/reflection.jvm/src/kotlin/reflect/jvm/internal/calls/InlineClassAwareCaller.kt

```diff
@@ -38,6 +40,16 @@ internal class InlineClassAwareCaller<out M : Member?>(
     override val parameterTypes: List<Type>
         get() = caller.parameterTypes
 
+    private val memberParameterTypes: List<Class<*>>
+        get() = when (val m = member) {
+            is Method -> m.parameterTypes.asList()
+            is Constructor<*> -> m.parameterTypes.asList()
+            is Field -> listOf(m.type)
+            // TODO: Check if there are any cases other than the above three, especially if they result in null.
+            // TODO: Check the Exceptions and error messages to be thrown.
+            else -> throw IllegalArgumentException("temp err")
```

Comment:
I have a question about the `member` property.

Looking at the `Javadoc` of the `Member` interface, it appears that this interface is not inherited by any classes other than `Method`, `Constructor` and `Field`.
https://docs.oracle.com/javase/jp/8/docs/api/java/lang/reflect/Member.html
In other words, this `else` clause is executed when the `member` property is `null`.

Now, is there a situation where the `member` property is `null` in `InlineClassAwareCaller`?
If present, modify the processing in `else`, and if not, modify the error message, respectively.